DAL/Repositories/UnitOfWork.cs

```diff
@@ -8,23 +8,27 @@ public class UnitOfWork : IUnitOfWork
         private readonly InternshipDbContext internshipDbContext;
 
         private IBaseRepository<Candidate> candidateRep;
+        private IBaseRepository<CandidateSandbox> candidateSandboxRep;
+        private IBaseRepository<City> cityRep;
+        private IBaseRepository<Country> countryRep;
         private IBaseRepository<Employee> employeeRep;
+        private IBaseRepository<EnglishLevel> englishLevelRep;
         private IBaseRepository<InternshipTeam> internshipTeamsRep;
         private IBaseRepository<Interview> interviewRep;
+        private IBaseRepository<Sandbox> sandboxRep;
         private IBaseRepository<Skill> skillRep;
-        private IBaseRepository<Topic> topicRep;
         private IBaseRepository<Specialization> specializationRep;
-        private IBaseRepository<EnglishLevel> englishLevelRep;
-        private IBaseRepository<Country> countryRep;
```

Comment:
please, do not move/exchange rows which are not related to your changes
in this case there can be merge conflicts
please review UnitOfWork.cs in the MAIN repo and make appropriate changes